app/controllers/users_controller.rb

```diff
@@ -6,8 +6,8 @@ def index
   end
 
   def show
-    user = User.find_by(username: params[:user])
-    @boards = user.boards.all
+    @user = User.find_by(username: params[:user])
+    @boards = @user.boards.all
```

Comment:
you can call boards on @user which will return all that users boards anyway so I think @boards is probably unnecessary.